spec/support/helper_methods.rb

```diff
@@ -16,17 +17,21 @@ def ignoring_warnings
   def with_safe_set_to_level_that_triggers_security_errors
     result = nil
 
+    orig_safe = $SAFE
     Thread.new do
       ignoring_warnings { $SAFE = SAFE_LEVEL_THAT_TRIGGERS_SECURITY_ERRORS }
       result = yield
     end.join
 
     # $SAFE is not supported on Rubinius
-    unless defined?(Rubinius)
-      expect($SAFE).to eql 0 # $SAFE should not have changed in this thread.
```

Comment:
I wonder what the point of this assertion was? It seems to be asserting a detail of implementation that isn't really that necessary to the four specs using the helper. Why wouldn't we just use the `ensure` method across the board (with maybe a note that it's technically not required on < 2.6?)?

Reply:
I'm fine with that.

Reply:
It's a guard when Ruby changed their mind, like Ruby 2.6 actually did...?

(but this didn't help: the following operations (e.g. File.expand_path) fails with SecurityError, so many following tests fail... I think it's okay to remove this.)